plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -234,19 +274,20 @@ public function test_disable_responsive_image_with_picture_element(): void {
 
 		$img_markup = apply_filters( 'the_content', $image );
```

Comment:
At this point, the IMG in `$img_markup` is a JPEG. Why isn't it a WebP? I think this method is missing `$this->mock_frontend_body_hooks();`.